package.json

```diff
@@ -39,10 +39,16 @@
     "vscode-debugprotocol": "^1.11.0"
   },
   "devDependencies": {
-    "fs-extra": "^0.30.0",
-    "tslint": "^3.14.0",
-    "typescript": "1.8.10",
-    "vscode": "^0.11.17"
+    "@types/diff": "0.0.31",
+    "@types/fs-extra": "0.0.35",
+    "@types/lodash": "^4.14.40",
```

Comment:
Yes, sure. http://austinpray.com/ops/2015/09/20/change-travis-node-version.html talks about how we can do that. And here is an example that does the same https://github.com/nodejs/nan/blob/master/.travis.yml